services/brig/src/Brig/CanonicalInterpreter.hs

```diff
@@ -276,19 +287,27 @@ runBrigToIO e (AppT ma) = do
               . mapError authenticationSubsystemErrorToHttpError
               . mapError teamInvitationErrorToHttpError
               . mapError userSubsystemErrorToHttpError
+              . mapError enterpriseLoginSubsystemErrorToHttpError
               . runNotificationSubsystemGundeck (defaultNotificationSubsystemConfig e.requestId)
               . runEvents
               . runDeleteQueue e.internalEvents
               . interpretPropertySubsystem propertySubsystemConfig
               . interpretVerificationCodeSubsystem
               . emailSubsystemInterpreter e.userTemplates e.teamTemplates e.templateBranding
+              . runEnterpriseLoginSubsystem
               . userSubsystemInterpreter
               . runTeamInvitationSubsystem teamInvitationSubsystemConfig
               . authSubsystemInterpreter
           )
     )
     $ runReaderT ma e
 
+mkEnterpriseLoginSubsystemConfig :: Env -> Maybe EnterpriseLoginSubsystemConfig
+mkEnterpriseLoginSubsystemConfig env = do
```

Comment:
i would have made a missing audit-email field an error, unless it is explicitly stated in the RFC that we should silently stop sending emails.

Reply:
The problem with that is that not all environments are going to have this feature. Still a valid remark!